types/react-table/react-table-tests.tsx

```diff
@@ -645,10 +645,10 @@ const Component = (props: {}) => {
                     // then sum any of those counts if they are
                     // aggregated further
                     aggregate: 'count',
-                    Aggregated: ({ cell: { value } }: CellProps<Data>) => `${value} Names`,
+                    Aggregated: ({ cell: { value } }: CellProps<Data>) => <>{value} Names</>,
```

Comment:
All fixes assume a function component. However, function components can only return JSX elements or null due to https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051

Reply:
So `ReactNode` is no longer a valid return type?  All of these used to return `string`, which is a valid `ReactNode` type.  Am I misunderstand this?

Reply:
> So ReactNode is no longer a valid return type?

It never was for function components due to TypeScript limitations. 

The valid type for `Aggregated` is `ComponentType | ReactNode`.
The reason this code was working is that `Aggregated` wasn't assigned to a function component but `{}` which we'll remove from `ReactNode`.